Source/Krypton Toolkit/Examples/CircularProgressBar/CircularProgressBar.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>net5.0-windows</TargetFramework>
+    <LangVersion>latest</LangVersion>
+    <UseWindowsForms>true</UseWindowsForms>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="krypton.toolkit.canary.lite" Version="6.0.2103.6" />
```

Comment:
If this is for Extended, then do not include the toolkit standard here.

Reply:
The issue is that I don't appear to have access to the extended toolkit controls from the toolbox, even if I build the project.